packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -83,7 +81,6 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
     this._calloutId = getId('callout');
     this.barChartSvgRef = React.createRef<SVGSVGElement>();
     this._emptyChartId = getId('_MSBC_empty');
-    this._barId = getId('_MSBC_rect_');
```

Comment:
getId is needed to maintain unique ids across different instances

Reply:
added back barId